package.json

```diff
@@ -31,6 +31,8 @@
     "form-data": "2.1.2",
     "form-serialize": "0.7.1",
     "hapi": "13.4.1",
+    "i18next": "^4.1.0",
+    "i18next-xhr-backend": "^1.2.1",
```

Comment:
These must be exact versions. Follow instructions in readme if you're using yarn to set the default prefix.